subprojects/language-groovy/src/integTest/groovy/org/gradle/groovy/GroovyDocIntegrationTest.groovy

```diff
@@ -112,39 +112,4 @@ class GroovyDocIntegrationTest extends MultiVersionIntegrationSpec {
         where:
         module << ['groovy']
     }
-
-    @Issue(["GRADLE-3174", "GRADLE-3463"])
-    def "Error in Groovydoc generation is logged"() {
```

Comment:
Hm, you're right. I kept the test running for older versions of Groovy. I don't think you can trigger the error since 2.4.8, because https://issues.apache.org/jira/browse/GROOVY-7340 was fixed.